### PR TITLE
Refactor/feedbacks

### PR DESCRIPTION
**Pull request by nargizasultan**

### CODE REVIEW

**Review by beksultancs — APPROVED** [review]
LGTM!

File: src/main/java/kg/gadgetarium/gadgetarium/dto/responses/FeedbackReadUnreadResponses.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class FeedbackReadUnreadResponses {
+    private String status;
```
Comment (ChyngyzIshen) [inline src/main/java/kg/gadgetarium/gadgetarium/dto/responses/FeedbackReadUnreadResponses.java:13]: Write in the next line pls

File: src/main/java/kg/gadgetarium/gadgetarium/db/model/FeedbackResponseStatus.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package kg.gadgetarium.gadgetarium.db.model;
+
+public enum FeedbackResponseStatus {
+
+  READ, UNREAD
```
Comment (ChyngyzIshen) [inline src/main/java/kg/gadgetarium/gadgetarium/db/model/FeedbackResponseStatus.java:5]: Write in separate lines pls

**Review by ChyngyzIshen — COMMENTED** [review]
Commented 06.09.2022 at 10.40 am

**Pull request merged**